Solutions/Corvus.Extensions.CosmosClient/Corvus/Extensions/Cosmos/EntityInstance{T}.cs

```diff
@@ -16,20 +16,14 @@ namespace Corvus.Extensions.Cosmos
     /// <remarks>This provides a serializable version of the response which decorates the entity with an <see cref="ETag"/>.</remarks>
     [JsonConverter(typeof(EntityInstanceJsonConverter))]
     public sealed class EntityInstance<T> : IEquatable<IEntityInstance<T>>, IEntityInstance<T>
+        where T : class
     {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="EntityInstance{T}"/> class.
-        /// </summary>
-        public EntityInstance()
-        {
-        }
-
```

Comment:
Removed the parameterless constructor to make nullability happy. This was previously required to do `Activator.CreateInstance(objectType)` in the JsonConverter but I've replaced with `Activator.CreateInstance(objectType, entity, etag)` to invoke the constructor below, as we know that the type is `EntityInstance<>` at that point.